frontend/src/components/ResultPopUp/ResultPopUp.css

```diff
@@ -49,12 +49,12 @@
   margin-top: 0.5rem;
 }
 
-.popUp__button:first-child {
+.popUp__button--first-child {
```

Comment:
It's ninja code, too close to a pseudo-element. You could just use the right pseudo-element rather than a class with a misleading name and a source of future bugs.